qcodes/station.py

```diff
@@ -113,7 +121,7 @@ class Station(Metadatable, DelegateAttributes):
     A user dict representing the YAML file that the station was loaded from"""
 
     def __init__(self, *components: Metadatable,
-                 config_file: Optional[str] = None,
+                 config_file: Optional[Union[str, Sequence[str]]] = None,
```

Comment:
should we consider making this input arg to be called `config_files`? Not sure how much extra work that would cause to change throughout the code base.

Reply:
I am not sure about this change as I think this is a breaking change. We can consider it when it absolutely becomes necessary and for now probably keeping the argument name same is ok.